src/map/skill.cpp

```diff
@@ -18954,28 +18960,12 @@ int skill_delunitgroup_(struct skill_unit_group *group, const char* file, int li
 		group->valstr = NULL;
 	}
 
-	idb_remove(skillunit_group_db, group->group_id);
-	map_freeblock(&group->unit->bl); // schedules deallocation of whole array (HACK)
```

Comment:
Did we add that somewhere else? I mean based on the comment it was hacky before, but did we do it better now?
Likewise should be in some destructor maybe?